src/main/java/racingcar/RacingCarController.java

```diff
@@ -0,0 +1,43 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCarController {
+
+    private Output output;
+    private List<Car> cars;
+
+    public RacingCarController(Output output, List<Car> cars) {
+        this.output = output;
+        this.cars = cars;
+    }
```

Comment:
매개변수로 받아 컨트롤러를 생성하기보다는
```suggestion
    public RacingCarController(Output output, List<Car> cars) {
        this.output = new Output;
        this.cars = new Cars;
    }
```
이런식으로 생성하는 생성하는게 나을 것 같아요